integration/client_sdk/go/auction/auction_test.go

```diff
@@ -0,0 +1,181 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+Copyright 2020 Intel Corporation
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hyperledger-labs/fabric-private-chaincode/client_sdk/go/fpc"
+	"github.com/hyperledger-labs/fabric-private-chaincode/integration/client_sdk/go/test_utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestChaincode(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Chaincode Suite")
+}
+
+var _ = Describe("Chaincode utils", func() {
+	var (
+		contract       fpc.Contract
+		auctionCounter int
+		auctionName    string
+		numClients     int
+	)
+
+	BeforeEach(func() {
+		network, err := test_utils.SetupNetwork("mychannel")
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(network).ShouldNot(BeNil())
+
+		//admin := fpc.GetManagementAPI(network, ccID)
+		//
+		//// Setup Chaincode Enclave
+		//attestationParams := []string{"some params"}
+		//err = admin.InitEnclave("peer0.peer1.example.com:7051", attestationParams...)
+		//Expect(err).ShouldNot(HaveOccurred())
+
+		// Get FPC Contract
+		ccID := "auction_test"
+		contract = fpc.GetContract(network, ccID)
+		Expect(contract).ShouldNot(BeNil())
+
+		res, err := contract.SubmitTransaction("init", "MyAuctionHouse")
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(res).Should(Equal([]byte("OK")))
+	})
+
+	Context("Scenario 1", func() {
+		When("close non existing auction.", func() {
```

Comment:
The scenarios below test mostly the auction chaincode, which is also tested in the  cli-based auction test, but not the client-sdk itself.  Might maybe be good to add also some scenario of testing client-sdk itself?  

Some additional scenarios which come in mind:
- a query (below are all invokes?)
- a query/tx on a non-existing chaincode
- a query/tx on an installed but not initialized (and hence not registered) chaincode

Reply:
I added test for the suggested scenarios.